### DISCUSSION TITLE
Feature Request: support for from clauses in graph

### DISCUSSION THREAD

**Discussion by marconms** [openfga/community#28] [Ideas]
I would like to see the support for `from` clauses introduced in the graph in some way.

They are a supported feature of OpenFGA, and they are used quite extensively in the documentation; their depiction would make a lot of sense in my opinion.

**Comment by rhamzeh** [comment]
Thanks for your suggestion @anvelicon!

We are working on improving the graph support in the Playground - some of the changes we're working on for an update to the DSL (needed for reverse expansion and listObjects optimization) will unblock some of the work we have planned for this feature.

Stay tuned!
